doc/source/dev/development_gitpod.rst

```diff
@@ -265,6 +265,35 @@ GitHub application page.
 .. image:: ./gitpod-imgs/gitpod-edit-permissions-repo.png
    :alt: Gitpod integrations - edit GH repository permissions screenshot
 
+
+Generic debugging/testing steps for gitpod environment. 
+******************************************************************************************
+
+A user might use the below steps one after another to debug/test Gitpod 
```

Comment:
Minor wording nit:
```suggestion
You can use the below steps to debug/test Gitpod 
```

Also, it would be nice to make capitalization consistent for Gitpod throughout the document.